centreon/src/Core/Service/Application/Repository/ReadServiceRepositoryInterface.php

```diff
@@ -108,6 +111,26 @@ public function findServiceNamesByHost(int $hostId): ?ServiceNamesByHost;
      */
     public function findById(int $serviceId): ?Service;
 
+    /**
+     * Find one service.
+     *
+     * @param int ...$serviceIds
+     *
+     * @throws \Throwable
+     *
+     * @return list<TinyService>
+     */
+    public function findByIds(int ...$serviceIds): array;
+
+    /**
+     * Find services based on given IDs.
+     *
+     * @throws \Throwable
+     *
+     * @return \Traversable<int, TinyService>&\Countable
+     */
+    public function findAll(): \Traversable&\Countable;
+
```

Comment:
Description of methods are not corresponding
```suggestion
    /**
     * Find services based on given IDs
     *
     * @param int ...$serviceIds
     *
     * @throws \Throwable
     *
     * @return list<TinyService>
     */
    public function findByIds(int ...$serviceIds): array;

    /**
     * Find all services
     *
     * @throws \Throwable
     *
     * @return \Traversable<int, TinyService>&\Countable
     */
    public function findAll(): \Traversable&\Countable;

```